### DISCUSSION TITLE
How to debug problematic call?

### DISCUSSION THREAD

**Discussion by Disregard-Therest** [serverpod/serverpod#1061] [Q&A] [answered]
Hi, after some changes I receive following messages

Failed call: doctor.changeSpecialty
ServerpodClientException: Internal server error. Call log id: 78, statusCode = 500

However, on local machine everything works fine.
How can I check the problem on the production server?
How can I see the aforementioned log id?
Is it possible to extend error processing so that I receive full info in Telegram for example?

**Comment by vlidholt** [comment, marked as answer]
Hi! This is what the Serverpod Insights app is for. It will connect to your server while it is running and allows you do read all the logs. :)

https://docs.serverpod.dev/insights

**Comment by Disregard-Therest** [comment]
Thank you, Viktor! Looks like a solution.

What should be done in order to establish connection? We put serverpod in docker, so, probably we need to forward the required port, right?

![image](https://github.com/serverpod/serverpod/assets/98102396/a7e1a40d-0dc9-4bb1-b395-262c56e82d81)

**Reply by Disregard-Therest** [reply]
![image](https://github.com/serverpod/serverpod/assets/98102396/090300e0-4d05-4491-af35-614d9c311c4a)

**Reply by Disregard-Therest** [reply]
something like this?

**Reply by Disregard-Therest** [reply]
no, it changes the port with every attempt

what should I config in order to establish connection?

**Reply by vlidholt** [reply]
By default Serverpod is running on port 8080, 8081, and 8082. You can change this in the config files e.g. `config/development.yaml`. The Serverpod Insights app also uses the yaml files to figure out how to connect to your server while it's running, so that needs to be correct too.

**Reply by Disregard-Therest** [reply]
ok, clear, thank you!
